docs/README.md

```diff
@@ -325,6 +325,42 @@ cd dashboard
 faas-cli deploy
 ```
 
+### Log storage with S3
+
+Generate secrets
```

Comment:
@kvuchkov Fancy trying to plumb this into the buildshiprun function to capture the logs?

Reply:
cc @s8sg @viveksyngh